Modelica/Blocks/Logical.mo

```diff
@@ -20,6 +20,20 @@ the output is <strong>false</strong>.
 </html>"));
   end And;
 
+  block MultiAnd "Logical and of Boolean vector elements"
+    extends Modelica.Blocks.Interfaces.PartialBooleanMISO;
+  equation
+    y = Modelica.Math.BooleanVectors.andTrue(u);
+    annotation (Icon(coordinateSystem(initialScale=0.1),
+      graphics={ Text(extent={{-90,40},{90,-40}},textString="and")}),
+      Documentation(info="<html>
+<p>
+The output is <strong>true</strong> if all elements of the input vector are <strong>true</strong>, otherwise
```

Comment:
I'm sorry, my bad.  It is `MultiNand` that still uses `allTrue`:
```
  block MultiNand "Logical nand of Boolean vector elements"
    extends Modelica.Blocks.Interfaces.PartialBooleanMISO;
  equation
    y=not Modelica.Math.BooleanVectors.allTrue(u);
```

Reply:
So, shouldn't we fix that so that Nand is the same as Not And?

Reply:
I'd love to see that kind of change, but can we do it this late in the release cycle considering that it is a bugfix on the border to a plain backwards incompatible change?

Reply:
> I'd love to see that kind of change, but can we do it this late in the release cycle considering that it is a bugfix on the border to a plain backwards incompatible change?

The models are new so we don't have to care about backwards compatibility, and to me it looks like MultiAnd was corrected, but not MultiNand.

Reply:
@AHaumer, could you please consider fixing `MultiNand` as part of this PR, so that we don't have to handle the annoying difference between `allTrue` and `andTrue` in the documentation?